src/plugins/weekSelect/weekSelect.ts

```diff
@@ -38,12 +38,16 @@ function weekSelectPlugin(): Plugin<PlusWeeks> {
         selDate.getMonth() === fp.currentMonth &&
         selDate.getFullYear() === fp.currentYear
       ) {
-        fp.weekStartDay = (<DayElement>fp.days.childNodes[
-          7 * Math.floor((<DayElement>fp.selectedDateElem).$i / 7)
-        ]).dateObj;
-        fp.weekEndDay = (<DayElement>fp.days.childNodes[
-          7 * Math.ceil((<DayElement>fp.selectedDateElem).$i / 7 + 0.01) - 1
-        ]).dateObj;
+        fp.weekStartDay = (<DayElement>(
+          fp.days.childNodes[
+            7 * Math.floor((<DayElement>fp.selectedDateElem).$i / 7)
+          ]
+        )).dateObj;
+        fp.weekEndDay = (<DayElement>(
+          fp.days.childNodes[
+            7 * Math.ceil((<DayElement>fp.selectedDateElem).$i / 7 + 0.01) - 1
+          ]
+        )).dateObj;
```

Comment:
the hunk above is some formatting change added by the husky pre-commit task only. :+1: